repotracker/repotracker.go

```diff
@@ -589,3 +595,67 @@ func createVersionItems(v *version.Version, ref *model.ProjectRef, project *mode
 	}
 	return nil
 }
+
+func addBuildBreakSubscriptions(v *version.Version, projectRef *model.ProjectRef) error {
+	subscriptionBase := event.Subscription{
+		Type:      event.ResourceTypeVersion,
+		Trigger:   "regression",
+		Selectors: trigger.MakeVersionSelectors(*v),
+	}
+	subscribers := []event.Subscriber{}
+
+	// if the commit author wants build break notifications, subscribe them to this commit
+	if v.AuthorID != "" {
+		subscriber, err := makeBuildBreakSubscriber(v.AuthorID)
+		if err != nil {
+			return err
```

Comment:
If this err is hit, go on to notify the project admins.

Reply:
done